app/cli/cmd/registered_integration_add.go

```diff
@@ -66,7 +66,9 @@ func newRegisteredIntegrationAddCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&integrationDescription, "description", "", "integration registration description")
-	cmd.Flags().StringSliceVar(&options, "options", nil, "integration arguments")
+	// StringSlice seems to struggle with comma-separated values such as p12 jsonKeys provided as passwords
+	// So we need to use StringArrayVar instead
+	cmd.Flags().StringArrayVar(&options, "opt", nil, "integration arguments")
```

Comment:
this has a consequence, that we no longer can provide things like `--opt foo=bar,bar=baz` but instead `--opt foo=bar --opt bar=baz`